src/qcodes/instrument/instrument_base.py

```diff
@@ -708,6 +719,9 @@ def get(self, param_name: str) -> Any:
         Returns:
             The current value of the parameter.
 
+        Note:
+            This is deprecated call get directly on the parameter.
```

Comment:
"This is deprecated. Call get directly on the parameter."
Also apply for the change below.